admin-dev/filemanager/dialog.php

```diff
@@ -424,12 +424,12 @@
 
     function filenameSort($x, $y)
     {
-        return $x['file'] < $y['file'];
+        return ($x['file'] < $y['file']) ? 1 : -1;
```

Comment:
I like this syntax better for these kind of function but totally optional :) (also, it can return `0` if `$x['file'] === $y['file']`)
```suggestion
        return $x['file'] <=> $y['file'];
```

Reply:
Oh, I didin't know this operator! That's great!